matter/wifi/wf200/host_if.cpp

```diff
@@ -230,9 +230,9 @@ sl_status_t sl_wfx_host_process_event(sl_wfx_generic_message_t *event_payload)
       sl_wfx_exception_ind_t *firmware_exception = (sl_wfx_exception_ind_t *)event_payload;
       uint8_t *exception_tmp                     = (uint8_t *)firmware_exception;
       EFR32_LOG("firmware exception\r\n");
-      for (uint16_t i = 0; i < firmware_exception->header.length; i += 16) {
+      for (uint16_t i = 0; i < firmware_exception->header.length; i += LENGTH_16) {
```

Comment:
why 16?

Reply:
need to check more details , kept LENGTH_16 only for now